content/foundations/icons/index.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Octicons
+---
+import {HeroLayout} from '@primer/gatsby-theme-doctocat'
+import Icons from '../../../src/components/icons'
+
+export default HeroLayout
```

Comment:
Should we use the HeroLayout here? Or just have a standard type heading on this page (Octicons) that says the current version 
<img width="714" alt="Screen Shot 2023-02-07 at 12 11 41 PM" src="https://user-images.githubusercontent.com/586552/217315437-9e279524-7e39-4cb5-b0c8-d89d623f3dbd.png">

Reply:
I copied it over the way it was, but I think we don't need the hero.

Reply:
Removed the hero ✅